sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_age_mismatch_inference.py

```
@@ -0,0 +1,159 @@
+        # Create author
+        author = models.DocumentAuthor(id="author2", full_name="authorName2")
+
+        create_date_time = datetime.datetime(2024, 2, 19, 0, 0, 0, 0, tzinfo=datetime.timezone.utc)
+        patient_document1 = models.PatientDocument(
+            type=models.DocumentType.NOTE,
+            clinical_type=models.ClinicalDocumentType.RADIOLOGY_REPORT,
+            id="doc2",
+            content=models.DocumentContent(source_type=models.DocumentContentSourceType.INLINE, value=doc_content1),
+            created_at=create_date_time,
+            specialty_type=models.SpecialtyType.RADIOLOGY,
+            administrative_metadata=models.DocumentAdministrativeMetadata(
+                ordered_procedures=[ordered_procedure], encounter_id="encounter2"
+            ),
+            authors=[author],
+            language="en",
+        )
+
+        # Construct patient
+        patient1 = models.PatientRecord(
+            id="patient_id2",
+            details=patient_info,
+            encounters=[encounter],
+            patient_documents=[patient_document1],
+        )
+
+        # Create a configuration
+        configuration = models.RadiologyInsightsModelConfiguration(verbose=False, include_evidence=True, locale="en-US")
+
+        # Construct the request with the patient and configuration
+        patient_data = models.RadiologyInsightsJob(
+            job_data=models.RadiologyInsightsData(patients=[patient1], configuration=configuration)
+        )
+
+        # Health Insights Radiology Insights
+        try:
+            poller = radiology_insights_client.begin_infer_radiology_insights(
+                id=job_id,
+                resource=patient_data,
+            )
+            radiology_insights_result = poller.result()
+            self.display_age_mismatch(radiology_insights_result, doc_content1)
+        except Exception as ex:
+            print(str(ex))
```

Shouldnt we just raise the exception here again?

---

applies to other samples